orchestrator/src/test/java/com/scottlogic/deg/orchestrator/cucumber/features/operators/general/EqualTo.feature

```diff
@@ -475,1436 +129,150 @@ Feature: User can specify that a value is equalTo a required value
       | foo |
       | 15  |
 
-  Scenario: 'EqualTo' a value and must be null should emit null
+  Scenario: 'EqualTo' a value and must be null should be contradictory
     Given there is a field foo
     And foo is equal to "a"
     And foo is null
-    Then the following data should be generated:
-      | foo  |
-      | null |
+    Then no data is created
 
 ### ofType ###
 
-  Scenario: 'OfType' string 'equalTo' a string value should be successful
+  Scenario Outline: 'EqualTo' should combine with type <type>
     Given there is a field foo
-    And foo is equal to "0123456789"
-    And foo is of type "string"
+    And foo is equal to <value>
+    And foo is of type <type>
     Then the following data should be generated:
-      | foo          |
-      | null         |
-      | "0123456789" |
+      | foo     |
+      | <value> |
+    Examples:
+      | type | value |
```

Comment:
Minor formatting thing - in a lot of these outlines the dividers are not aligned and it looks clesner/is more parseable when they are